ui/app/helpers/utils/token-util.js

```diff
@@ -217,6 +217,10 @@ export function getTokenFiatAmount(
   const currentTokenToFiatRate = multiplyCurrencies(
     contractExchangeRate,
     conversionRate,
+    {
+      multiplicandBase: 10,
+      multiplierBase: 16,
+    },
```

Comment:
I'm still trying to figure out `contractExchangeRate`, but the base of `conversionRate` is definitely `10`.

Reply:
Update: both should be `10`. The `contractExchangeRate` comes from this data source (via the TokenRatesController): https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48&vs_currencies=eth